src/util/ReactUtils.ts

```diff
@@ -110,27 +110,28 @@ export const findAllByType = (
     types = [getDisplayName(type)];
   }
 
-  toArray(children).forEach((child: React.DetailedReactHTMLElement<any, HTMLElement>) => {
+  toArray(children).forEach(child => {
     const childType = _.get(child, 'type.displayName') || _.get(child, 'type.name');
     if (types.indexOf(childType) !== -1) {
-      result.push(child);
+      result.push(child as DetailedElement);
     }
   });
 
   return result;
-};
+}
+
 /*
  * Return the first matched child by type, return null otherwise.
  * `type` can be a React element class or string.
```

Comment:
Same here, comment needs updated. String no longer accepted and curious if function components get desired results.